search/src/main/java/edu/artic/search/SearchBaseFragment.kt

```diff
@@ -40,7 +38,19 @@ abstract class SearchBaseFragment<TViewModel : SearchBaseViewModel> : BaseViewMo
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        val lm = GridLayoutManager(view.context, 5)
+        var width = resources.displayMetrics.widthPixels
+        val ten = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 10f, resources.displayMetrics).toInt()
```

Comment:
There's now an extension method to simplify the conversion process. Please use `edu.artic.base.utils.dpToPixels` for this and the other DIP values.